bridge/client/app/_components/ktb-loading-spinner/ktb-loading-spinner.component.scss

```diff
@@ -0,0 +1,84 @@
+@import '~@dynatrace/barista-components/core/src/style/variables';
+
+$ktb-loading-spinner-default-size: 30px;
+$ktb-loading-spinner-small-size: 15px;
+$ktb-loading-spinner-offset: 22px * 3.1416; // r(-gap)*2*3.14
+$ktb-loading-spinner-duration: 1700ms;
+$ktb-loading-spinner-ease: cubic-bezier(0.645, 0.045, 0.355, 1);
+
+:host {
+  display: inline-block;
+  width: $ktb-loading-spinner-default-size;
+  height: $ktb-loading-spinner-default-size;
+}
+
+// There is an issue in angular with the combination of :host ans :host-context.
+// As a workaround you have to move the :host-context after the :host selector
+:host :host-context(.dt-form-field),
+:host :host-context(.dt-filter-field),
+:host :host-context(.dt-button) {
+  width: $ktb-loading-spinner-small-size;
+  height: $ktb-loading-spinner-small-size;
+
+  // This overrides the min-with requirement for prefix/suffix
+  min-width: $ktb-loading-spinner-small-size;
+}
+
+.ktb-loading-spinner-svg {
+  display: block;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  stroke-width: 4px;
+  stroke: $logolime;
```

Comment:
This results in my test to `#b4dc00`, which is not the blue color from the screenshot in the PR.
But, I would suggest to use here css variables and retrieve the color via e.g. `var(--my-loading-spinner-color);` so that this color can be changed via custom theme.

Reply:
Thanks for mentioning this, I've changed this in the latest commit :+1: